latex/programming_interface.tex

```diff
@@ -189,29 +190,29 @@ \subsection{Common by-value semantics}
 \addFootNotes{Common special member functions for by-value semantics}
 {table.specialmembers.common.byval}
   \addRow
-    {T(const T \&rhs) = default}
+    {T(const T \&rhs);}
     {
-      Default copy constructor.  
+      Copy constructor.
```

Comment:
Perhaps we should also state here that these member functions can be implicit or explicitly defined as defaulted.

Reply:
If we are following the rule of zero, it is incorrect for all five of these special member functions to be explicitly defined as defaulted and inlined.

Reply:
We had already a similar discussion in the past about whether the SYCL spec should describe the real implementation or just some equivalent behaviour (such as "there are some constructors, synthesized or real, that behaves like if they where defaulted...").
For example since triSYCL relies aggressively on very modern C++, some constructors are not implemented at all, just synthesized and thus probably departs from the spec because of this...

Reply:
A reference to the rule of five and rule of zero has been added to the table.